test/general/__init__.py

```diff
@@ -27,3 +28,51 @@ def setup_solo_multiworld(world_type: Type[World], steps: Tuple[str, ...] = gen_
     for step in steps:
         call_all(multiworld, step)
     return multiworld
+
+
+def generate_multiworld(players: int = 1) -> MultiWorld:
```

Comment:
Might just docstring it tbh, the whole point of moving it from test_fill was other tests wanted it so docs are probably appropriate in case even more want it in the future